indexer/processors/contracts_bedrock/cross_domain_messenger.go

```diff
@@ -0,0 +1,167 @@
+package contracts_bedrock
```

Comment:
```suggestion
package contracts
```
Can just name them contracts because contracts_bedrock is suposed to be contracts. It's only contracts_bedrock to avoid name collision and they plan on changing it eventually

Reply:
+1 to just naming them contracts. I'll update